lib/DidKeyDriver.js

```diff
@@ -50,9 +51,16 @@ export class DidKeyDriver {
    *   with the corresponding key pairs used to generate it (for storage in a
    *   KMS).
    */
-  async generate({seed} = {}) {
+  async generate({seed, ...keyPairOptions} = {}) {
     // Public/private key pair of the main did:key signing/verification key
-    const verificationKeyPair = await this.verificationSuite.generate({seed});
+    let verificationKeyPair;
+    if(this.verificationSuite === EcdsaMultikey) {
```

Comment:
Hmm, so I was expecting a verificationSuite adapter API to be exposed that could wrap a multikey lib -- and then that would be passed into the constructor. The way this is written, it will work when the exact same version of the ecdsa multikey library used here matches with what the application uses. It would be better to either expose verificationSuite adapter utility API ... or make the constructor also accept `generate` and `from` methods ... or a `multikeyApi` or something.

Given that we need further adjustments to the API (#53), let's just expose a verification suite adapter utility function:

`createVerificationSuite({generate, from})` ... that we can pass `EcdsaMultikey` (or another multikey lib) into to create a verification suite. It should return a verification instance that is doing what's shown below in these if/else statements -- and then those can be simplified back to what they used to be.

Reply:
Updated to use adapter function.